.github/workflows/pullrequests.yml

```diff
@@ -0,0 +1,59 @@
+name: flask-rebar Pull Request Tests
+
+on:
+- pull_request
+
+jobs:
+  tests:
+    name: Testing on Python ${{ matrix.python-version }} and ${{ matrix.os }} 
```

Comment:
Looks like this should be `{{ matrix.python }}` (and `matrix.os` is always blank so we could just leave that off)?